cpp/tests/io/nested_json_test.cu

```diff
@@ -14,17 +14,121 @@
  * limitations under the License.
```

Comment:
can this file be .cpp?

Reply:
Mh, don't think so, as we have some cuda calls in there, like `cudaMemcpyAsync`

Reply:
if you have the right include paths set (`target_link_libraries(... CUDA::...)`) and include cuda.h, it should still work?

Reply:
After thinking about it, that makes sense. Thanks!